sample/src/main/java/com/google/accompanist/sample/webview/WebViewSaveStateSample.kt

```diff
@@ -0,0 +1,114 @@
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.google.accompanist.sample.webview
+
+import android.os.Bundle
+import androidx.activity.ComponentActivity
+import androidx.activity.compose.setContent
+import androidx.compose.foundation.layout.Column
+import androidx.compose.foundation.layout.Row
+import androidx.compose.foundation.layout.Spacer
+import androidx.compose.foundation.layout.fillMaxSize
+import androidx.compose.foundation.layout.size
+import androidx.compose.material.Button
+import androidx.compose.material.MaterialTheme
+import androidx.compose.material.Surface
+import androidx.compose.material.Text
+import androidx.compose.runtime.Composable
+import androidx.compose.runtime.LaunchedEffect
+import androidx.compose.runtime.mutableStateOf
+import androidx.compose.runtime.saveable.rememberSaveable
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.unit.dp
+import androidx.navigation.compose.NavHost
+import androidx.navigation.compose.composable
+import androidx.navigation.compose.rememberNavController
+import com.google.accompanist.web.WebView
+import com.google.accompanist.web.rememberSaveableWebViewState
+import com.google.accompanist.web.rememberWebViewNavigator
+import com.google.accompanist.web.rememberWebViewState
+
+class WebViewSaveStateSample : ComponentActivity() {
+    override fun onCreate(savedInstanceState: Bundle?) {
+        super.onCreate(savedInstanceState)
+        setContent {
+            MaterialTheme {
+                Surface {
+                    val navController = rememberNavController()
+                    Column(Modifier.fillMaxSize()) {
+                        Row {
+                            Button(onClick = { navController.popBackStack() }) {
+                                Text("Home")
+                            }
+                            Button(onClick = { navController.navigate("detail") }) {
+                                Text("Detail")
+                            }
+                        }
+
+                        Spacer(modifier = Modifier.size(16.dp))
+
+                        NavHost(navController = navController, startDestination = "home") {
+                            composable("home") {
+                                Home()
+                            }
+                            composable("detail") {
+                                Detail()
+                            }
+                        }
+                    }
+                }
+            }
+        }
+    }
+}
+
+@Composable
+private fun Home() {
+    val webViewState = rememberSaveableWebViewState()
+    val webViewBundleState = rememberSaveable { mutableStateOf<Bundle?>(null) }
```

Comment:
I think it'd be better if this state was handled internally by the `WebView` wrapper, and then it'd function like other components that have some sort of `rememberSaveable` internal state. A wrapping `key` would be able to reset the state if desired.